package.json

```diff
@@ -21,22 +23,31 @@
   "dependencies": {
     "handlebars": "^4.7.8",
     "inquirer": "^9.2.16",
+    "package1": "^0.0.0",
```

Comment:
@shravan20 Kindly remove this package1 and 2 , hope it's not needed

Reply:
My bad, this is coming due to test cases. Will fix this.